test/integration/pkg/client/msp/identity_test.go

```diff
@@ -216,6 +216,46 @@ func TestGetAllIdentities(t *testing.T) {
 	}
 }
 
+func TestSigningIdentityPrivateKey(t *testing.T) {
+	mspClient, sdk := setupClient(t)
+	defer integration.CleanupUserData(t, sdk)
+
+	// Generate a random user name
+	username := integration.GenerateRandomID()
+
+	req := &msp.IdentityRequest{
+		ID:          username,
+		Affiliation: "org2",
+		Type:        IdentityTypeUser,
+	}
+
+	// Create new identity
+	newIdentity, err := mspClient.CreateIdentity(req)
+	if err != nil {
+		t.Fatalf("Create identity failed: %s", err)
+	}
+
+	if newIdentity.Secret == "" {
+		t.Fatal("Secret should have been generated")
+	}
+
+	// Enroll the new user
+	err = mspClient.Enroll(username, msp.WithSecret(newIdentity.Secret))
+	if err != nil {
+		t.Fatalf("Enroll failed: %s", err)
+	}
+
+	// Get the new user's signing identity
+	si, err := mspClient.GetSigningIdentity(username)
+	if err != nil {
+		t.Fatalf("GetSigningIdentity failed: %s", err)
+	}
+
+	if _, err := si.PrivateKey().Bytes(); err != nil {
```

Comment:
It would be good to verify that Bytes() actually returns correct value. Perhaps try to use it in some way.

Reply:
Fair enough, please view the updated integration test here:  https://github.com/pplavetzki/fabric-sdk-go/blob/ec4852b930745b39acae60af4dcb36534908872c/test/integration/pkg/client/msp/identity_test.go#L261

Cheers!